### DISCUSSION TITLE
Is the `ask` kitten supposed to return JSON?

### DISCUSSION THREAD

**Discussion by salmankhilji** [kovidgoyal/kitty#5919] [Q&A] [answered]
Entering `hello` in `kitty +kitten ask` returns:

```
{'items': [], 'response': 'hello'}
```

`jq` can not process the response.  After looking it up online, if the strings are enclosed using single quotes, then the JSON is an invalid JSON.

Before I file a ticket, the the response from the `ask` kitten supposed to be JSON?

**Comment by kovidgoyal** [comment, marked as answer]
Not really, this is another "internal use" kitten, but no reason it cant
be made to valid output JSON, done.
